protocol/messenger.go

```diff
@@ -4154,6 +4154,11 @@ func (m *Messenger) MarkAllRead(ctx context.Context, chatID string) error {
 	return m.markAllRead(chatID, clock, true)
 }
 
+func (m *Messenger) MarkReadCommunityOverview(ctx context.Context, communityID string) error {
```

Comment:
we can just remove this function and call `DismissAllActivityCenterNotificationsFromCommunity` directly?

Reply:
@qfrank that function will dismiss notification related to community channel's but we need another function for dismiss notification related to just community it self and not it's channels.

Reply:
oh, sorry, i mean `m.DismissActivityCenterNotificationsCommunity` , not `DismissAllActivityCenterNotificationsFromCommunity` @mohsen-ghafouri

Reply:
we don't have `DismissActivityCenterNotificationsCommunity` func @qfrank

Reply:
emm.. isn't this one? https://github.com/status-im/status-go/blob/2410c01ab72122ecd20a838a14cf63e3e78ed141/protocol/messenger_activity_center.go#L450

Reply:
in `develop` branch we don't have it

Reply:
Oh, i saw you already removed the changes to `protocol/messenger.go`, that's what i mean, thank you!